stubs/fpdf2/fpdf/fpdf.pyi

```diff
@@ -451,17 +466,15 @@ class FPDF:
         title: str | None = None,
         alt_text: str | None = None,
         dims: tuple[float, float] | None = None,
+        keep_aspect_ratio: bool = False,
     ) -> _Image: ...
+    def preload_image(self, name: str | Image.Image | BytesIO, dims: Incomplete | None = None) -> tuple[str, Any, ImageInfo]: ...
```

Comment:
The docstring says:

>dims (Tuple[float]): optional dimensions as a tuple (width, height) to resize the image
>    before storing it in the PDF.

https://github.com/PyFPDF/fpdf2/blob/615db585a53dde9af5874fb0a58f33ee7f84aa2b/fpdf/fpdf.py#L3821-L3822

```suggestion
    def preload_image(self, name: str | Image.Image | BytesIO, dims: tuple[float, float] | None = None) -> tuple[str, Any, ImageInfo]: ...
```